gravitee-apim-repository/gravitee-apim-repository-mongodb/src/test/java/io/gravitee/repository/mongodb/MongoTestRepositoryConfiguration.java

```diff
@@ -46,6 +50,9 @@
     public MongoDBContainer mongoDBContainer() {
         MongoDBContainer mongoDb = new MongoDBContainer(DockerImageName.parse("mongo:" + mongoVersion));
         mongoDb.start();
+
+        LOG.info("MongoDB version: {}", getMongoFullVersion(mongoDb.getContainerInfo().getConfig().getEnv()));
```

Comment:
or maybe it could have been added directly on the log line thanks to `logback-test.xml` ? 🤔

Reply:
I tried a few things after reading https://stackoverflow.com/a/24971553/4069646 and wasn't super happy with the result, I think I will merge the PR like this and we can later improve that